frappe/templates/includes/comments/comments.py

```diff
@@ -3,29 +3,45 @@
 from __future__ import unicode_literals
 
 import frappe
-import frappe.utils
+import re
 from frappe.website.render import clear_cache
+from frappe.utils import add_to_date, now
 
 from frappe import _
 
-@frappe.whitelist(allow_guest=True)
+@frappe.whitelist()
 def add_comment(comment, comment_email, comment_by, reference_doctype, reference_name, route):
 	doc = frappe.get_doc(reference_doctype, reference_name)
 
-	if len(comment) < 10:
-		frappe.msgprint(_('Comment Should be atleast 10 characters'))
-		return ''
+	if not comment.strip() or len(comment) < 10:
+		frappe.msgprint(_('Your comment should be atleast 10 characters long'))
+		return False
```

Comment:
Why do we have this restriction? 

comments like "Thanks", "Great", "Inspiring" can be valid right?